tsl/src/compression/api.c

```diff
@@ -317,6 +318,11 @@ find_chunk_to_merge_into(Hypertable *ht, Chunk *current_chunk)
 	if (!ts_compression_settings_equal(ht_comp_settings, prev_comp_settings))
 		return NULL;
 
+	/* We don't support merging chunks with sequence numbers */
+	if (get_attnum(prev_comp_reloid, COMPRESSION_COLUMN_METADATA_SEQUENCE_NUM_NAME) !=
+		InvalidAttrNumber)
+		return NULL;
```

Comment:
do we need an error message here?

Reply:
No, we just disable chunks with seq numbers from being merged, they will be compressed as individual chunks.